src/lib/utils.tsx

```diff
@@ -69,3 +73,139 @@ export const useProjectId = (props: { projectId?: string }) => {
 
   return projectId;
 };
+
+/**
+ * 
+ * @param data
+ * text data
+ * @param count 
+ * the variable number used to determine which plural case to use
+ * zero = 0
+ * one = 1
+ * two = 2
+ * few = 3,4,5
+ * many = 6,7,...,99
+ * other = 100, 101, ...
+ * if count is provided but not matching plural key, fallback to base plural value
+ * in future we should also user's to define their own middleware for picking plurals
```

Comment:
nit: would be helpful in this jsdoc to have a link to the i18n specification that all of the different count values were pulled from